vite.config.ts

```diff
@@ -9,4 +9,7 @@ export default defineConfig({
     sourcemap: true,
     manifest: true,
   },
+  define: {
+    'process.env': process.env,
```

Comment:
@therealparmesh Lemme know if there's a more preferable way to do this in vite-land.

Reply:
I believe Vite already provides these variables within `import.meta`. Check this out: https://vitejs.dev/guide/env-and-mode.html#env-variables

FWIW, this seems to be closer to standards than Webpack's `process.env`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import.meta

Reply:
The tricky part is we're using a library (as a transitive dependency) which references `process.env` assuming its variables are string replaced at build-time (which isn't the case w/ Vite), so we need to inject an empty `process.env` object to unbreak a runtime error thrown in that library. Not sure if there's a better way to do that though.

[This section](https://github.com/ChainSafe/web3.js/blob/202be5b9ab848bc25f3a9948fe7dc84c7e57a66f/README.md#trouble-shooting-and-known-issues) of the library README alludes to a similar issue which happens with Angular 11 builds.

Reply:
Ah I see what you mean! Then yeah, this is the best way to go.

Reply:
Doh! I should've read the PR description first.

As a side note: Those last 2 points are definitely worth fixing in that library.